app-emulation/wa-linux-agent/wa-linux-agent-2.1.3.ebuild

```diff
@@ -1,28 +1,28 @@
 # Copyright (c) 2014 CoreOS, Inc.. All rights reserved.
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-
-MY_PN="WALinuxAgent"
-MY_PV="WALinuxAgent-${PV}"
-MY_P="${MY_PN}-${MY_PV}"
-S="${WORKDIR}/${MY_P}"
+EAPI=6
 
 DESCRIPTION="Windows Azure Linux Agent"
 HOMEPAGE="https://github.com/Azure/WALinuxAgent"
 KEYWORDS="amd64"
-SRC_URI="${HOMEPAGE}/archive/${MY_PV}.tar.gz"
+SRC_URI="${HOMEPAGE}/archive/v${PV}.tar.gz"
```

Comment:
Please use `SRC_URI="${HOMEPAGE}/archive/v${PV}.tar.gz -> ${P}.tar.gz"` because we shouldn't mirror a tarball simply named "v2.1.3.tar.gz"